src/utils/valid_input_reg_exp.test.ts

```diff
@@ -0,0 +1,41 @@
+import { expect } from 'chai';
+import isValidInput from './valid_input_reg_exp';
+
+const invalidValues = [
```

Comment:
I would also add `'.'` to the invalid values list.

Reply:
This RegExp is run when the user enters a value, I want them to be able to type a `'.'` so they can add a decimal